src/test/java/com/abdulraham/taxipassenger/addUserTest.java

```diff
@@ -0,0 +1,62 @@
+package com.abdulraham.taxipassenger;
+
+import com.abdulraham.taxipassenger.model.User;
+import com.abdulraham.taxipassenger.services.UserServices;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class addUserTest {
+
+    @Test
+    void check_add_user(){
+        UserServices userServices = new UserServices();
+        User user = new User();
+        user.setFirstName("abed");
+        user.setLastName("dahab");
+        user.setEmail("abood@hotmail.com");
+        user.setPhoneNum("0955227410");
+
+        boolean isAdded = userServices.isUserAdded(user);
```

Comment:
you don't need to define new variable. you can just do : 
assertTrue(userServices.isUserAdded(user));